docs/develop/guides/cheat-sheets/momento-cache-nodejs-cheat-sheet.md

```diff
@@ -0,0 +1,182 @@
+---
+sidebar_position: 3
+sidebar_label: TypeScript+Node.js
+sidebar_class_name: sidebar-item-nodejs
+title: Cheat Sheet for TypeScript and Node.js
+description: Get started really quickly coding using TypeScript and Node.js with Momento Cache
+---
+
+import Tabs from '@theme/Tabs';
+import TabItem from '@theme/TabItem';
+
+# Cheat Sheet for Node.js with Momento Cache
+If you need to get going quickly with Node.js + TypeScript and Momento Cache, this page contains the basic API calls you'll need. [Check the node.js SDK page](https://github.com/momentohq/client-sdk-nodejs) for this code in a full file format.
+
+## Install the Momento client library
+
+Install the client library in an existing node.js project:
+
+```cli
+npm install @gomomento/sdk
+```
+
+## Create a .env file
+You'll need to create a .env file to hold your Momento Cache auth token and the TTL (in seconds) you want to use by default.
+
+```
+export MOMENTO_AUTH_TOKEN=<your Momento token here>
+export MOMENTO_TTL_SECONDS=300
+```
+
+## Import libraries and connect to return a CacheClient object
+This code sets up the main function, the necessary imports,  and the CacheClient instantiation that each of the other functions will need to call.
+
+  <Tabs>
+    <TabItem value="ts" label="TypeScript" default>
+
+  ```typescript
+  import {
+    CacheGet,
+    CreateCache,
+    CacheSet,
+    CacheClient,
+    Configurations,
+    CredentialProvider,
+  } from '@gomomento/sdk';
+
+  import dotenv from 'dotenv';
+
+  dotenv.config();
+
+  async function main() {
+    const momento = new CacheClient({
+      configuration: Configurations.Laptop.v1(),
+      credentialProvider: CredentialProvider.fromEnvironmentVariable({
+        environmentVariableName: 'MOMENTO_AUTH_TOKEN',
+      }),
+      defaultTtlSeconds: 60,
+    });
+
+  main()
+    .then(() => {
+      console.log('success!!');
+    })
+    .catch((e: Error) => {
+      console.error(`Uncaught exception while running example: ${e.message}`);
+      throw e;
+    });
+  ```
+
+  </TabItem>
+  <TabItem value="nodejs" label="Node.js" default>
+    Coming soon.
+  </TabItem>
```

Comment:
would nodjs look very different? just wondering if can just have single tab instead seperate typescript and nodejs ones. I feel like would look identical. Fine to wait can tackle that in later PR's though